test/testcafe/spec/PollView_spec.js

```diff
@@ -55,6 +55,6 @@ test.requestHooks(requestLogger, identifyPollErrorMock)('not poll on error', asy
   await identityPage.fillPasswordField('password');
   await identityPage.clickNextButton();
   const pollingPageObject = new PollingPageObject();
-  await t.expect(pollingPageObject.getErrorMessages().getTextContent()).eql('Something went wrong, Try again later.');
+  await t.expect(pollingPageObject.getErrorMessages().getTextContent()).eql('Server is unable to respond at the moment.');
```

Comment:
terminal-polling-window-expired.json and error-safe-mode-polling.json both have the same i18n error key but different messages. I picked the better one.

Reply:
this was the error message returned by server. Might have to test if it breaks any selenium, hopefully should not because we may have not coverage for pollview..

Reply:
I can't find either of these strings in monolith 🧐

Reply:
Interesting. we are good then

Reply:
I found `E0000010` mapped to  `Service is in read only mode` in the test mocks in monolith, which is what we have for the key `errors.E0000010`. Oy vey.